src/main/java/org/opentripplanner/middleware/utils/ConfigUtils.java

```diff
@@ -26,13 +26,31 @@
 
     private static JsonNode envConfig;
 
+    /**
+     * Returns true only if an environment variable exists and is set to "true".
+     */
+    public static boolean getBooleanEnvVar(String var) {
+        String variable = System.getenv(var);
+        return variable != null && variable.equals("true");
+    }
+
+    /**
+     * Check if running in Travis CI. A list of default environment variables from Travis is here:
+     * https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
+     */
+    public static boolean isRunningCi() {
+        return getBooleanEnvVar("TRAVIS") && getBooleanEnvVar("CONTINUOUS_INTEGRATION");
+    }
+
     /**
      * Load config files from either program arguments or (if no args specified) from
      * default configuration file locations. Config fields are retrieved with getConfigProperty.
      */
     public static void loadConfig(String[] args) throws IOException {
         FileInputStream envConfigStream;
-
+        if (isRunningCi()) {
```

Comment:
All these checks could use a comment in the method they are contained in to note the use of environment variables when running on Travis.